src/main/java/duke/task/ToDo.java

```diff
@@ -0,0 +1,29 @@
+package duke.task;
+
+import java.time.LocalDate;
+
+public class ToDo extends Task {
+
+    public ToDo(String description, boolean isDone) {
+        super(description, isDone);
+    }
+
+    public String toString() {
+        String icon = this.completed ? "[" + "\u2713" + "]" : "[" + "\u2718" + "]";
+        return "[T]" + icon + " " + this.description;
+    }
+
+    /**
+     * Encode duke.task into a String to be saved in text file.
+     *
+     * @return String of encoded duke.task details.
+     */
+    public String toEncoding() {
+        int completedBinary = this.completed ? 1 : 0;
+        return "T>" + completedBinary + ">" + this.description;
+    }
```

Comment:
Maybe it's better to add the @Override keyword? So that others know that it is overriding an abstract method defined in the superclass.